Configuration/AlCa/python/autoCond.py

```diff
@@ -10,17 +10,19 @@
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run1
     'run1_mc_pa'        :   '93X_mcRun1_pA_v0',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
-    'run2_design'       :   '93X_mcRun2_design_v0',
+    'run2_design'       :   '93X_mcRun2_design_v1',
     # GlobalTag for MC production with pessimistic alignment and calibrations for Run2
-    'run2_mc_50ns'      :   '93X_mcRun2_startup_v0',
+    'run2_mc_50ns'      :   '93X_mcRun2_startup_v1',
     #GlobalTag for MC production with optimistic alignment and calibrations for Run2
-    'run2_mc'           :   '93X_mcRun2_asymptotic_v1',
+    'run2_mc'           :   '93X_mcRun2_asymptotic_v2',
+    # GlobalTag for MC production (L1 Trigger Stage1) with starup-like alignment and calibrations for Run2, L1 trigger in Stage1 mode
+    'run2_mc_l1stage1'  :   '93X_mcRun2_asymptotic_v1',
```

Comment:
@rekovic  @Martin-Grunewald we can not support an extra key to support the stage1 and stage 2 separately because this costs us a lot as we have to maintain another queue with stage 1 configurations. There must be another way out.

Reply:
@arunhep 
Stage1 is a different detector from Stage2. It uses different conditions. 

If we don't want to support an extra key for Stage1, we can then chose to configure Stage1 from static files (i.e. use hackConditions) and have Stage2 be configured from DB.
I don't mind this solution too much, as Stage1 is past, and its conditions are not going to
be changing.  As long as Stage2 in configurable from DB, I would be happy.  And AlCa ?

Reply:
I think it may not be ideal but a reasonable approach to support stage1 (past) with static files and Stage2 from DB.

Reply:
@arunhep 
Thinking of it further, mind you that we would not have to maintain the ```run2_mc_l1stage1``` key at all.  It will not be changing, as the conditions for a past detector will not evolve, so the key will always point to the same GT in the DB.  As we should keep the conditions in the DB, and not in static files, this PR might be the optimal solution.